src/test/java/core/basesyntax/dao/impl/StorageTest.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax.dao.impl;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.storage.Storage;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class StorageTest {
+    private StorageDao storage = new StorageDaoImpl();
```

Comment:
initialize in method annotated BeforeClass or Before (depends on junit version) 
the point is that the method will be executed just once